### DISCUSSION TITLE
Version upgrade to v1.0-rel0

### DISCUSSION THREAD

**Discussion by bokdeuk-jeong** [islet-project/islet#412] [Ideas]
The most recent version of qemu-rme at [Building an RME stack for QEMU - Enable Arm Architecture in QEMU - Confluence](https://linaro.atlassian.net/wiki/spaces/QEMU/pages/29051027459/Building+an+RME+stack+for+QEMU) is based on CCAv1.0-rel0.

As a part of some tests to see if our current Islet, which is based on 1.0-eac5, can be running on the current qemu-rme(v4), I ran requried sw , tf-rmm(v3) with a minor change, kvmtool(v2) and linux(v4), which seem to be for versions lower than 1.0-rel0.
I managed to run tf-rmm and the host, but failed to launch a realm with an error below:
`KVM_CAP_RME(KVM_CAP_ARM_RME_CONFIG_REALM) hash_algo: Invalid argument`.

So rather than figuring out the proper combination of sw stack, I propse to upgrade Islet to v1.0-rel0. One way or the other, we will need this soon. 
While some work on the version upgrade, others may use islet with 1.0-eac5 on fvp or tf-rmm with 1.0-rel0 on qemu.
I expect this shouldn't take long--at mot, a month.

The scope of work is as follows:

1. First, implement RMIs and RSIs according to the spec v1.0-rel0.
2. Second, update the implementation reflecting the defect resolution the spec in 1.1.
3. Third, platform spefic implementation for qemu.

Please share your thoughts on this.

**Comment by bokdeuk-jeong** [comment]
@p-sawicki2 , @Havner , @L0czek , @hihi-wang , @zpzigi754 , @nook1208

**Comment by nook1208** [comment]
> I propse to upgrade Islet to v1.0-rel0.

I agree. Because sooner or later we should update islet to meet the latest CCA spec

**Comment by p-sawicki2** [comment]
There are small changes in the attestation token format concerning how the RAK key is encoded. In the scope of our project we'll have to adjust `rust-rsi`. As to the other components that we're depend on, at first glance it seems that Veraison already provides support for this new token format: https://github.com/veraison/ccatoken/commit/b8e6d268dfc92f5efb3c96d8bed41694b288d4b4
https://github.com/veraison/rust-ccatoken/commit/f3c356398b275a86ed214b328ff8fc309490a806
